app/src/views/Product.vue

```diff
@@ -287,12 +298,13 @@ export default class Product extends Vue {
 
   get queryTags(): string[] | null {
     // If no selection, consider them all checked
-    const noneChecked = this.filters.categories.every((c) => !c.checked);
-    if (noneChecked) {
+    const noneCategoryChecked = this.filters.categories.every((c) => !c.checked);
+    const noneExpertiseChecked = this.filters.expertise_level.every((c) => !c.checked);
+    if (noneCategoryChecked && noneExpertiseChecked) {
       return null;
     }
 
-    return this.filters.categories.filter((x) => x.checked).map((x) => x.value);
+    return this.filters.categories.filter((x) => x.checked).map((x) => x.value).concat(this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value));
```

Comment:
`this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value)`
This actually needs to go somewhere else. In queryParams(), we need to add to the where attribute and tell it to look at "metadata.expertise"

something like the following
```
  const expertises = this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value);
  if (chosenExpertises.length > 0) {
      q.where = [...q.where,
        {
          fieldPath: "metadata.expertise",
          operator: "array-contains-any",
          value: chosenExpertises,
        },
      ];
    }
 ```
 
 That doesn't handle when tags aren't queried as well, I think, so you'll want to handle the nulls and stuff as well. (maybe default q.where to empty array? that would do it)

Reply:
Hey billy, I tried this approach of  ``` q.where = [...q.where, ``` 
It deosnt work. I also couldn't find anyother approach to adding 2 fieldpath into a single query. 

Can you help here? 
The last option I can think of is to create a separate @watcher method to do a 2nd query on 'expertise' feildPath and leave the current @watcher method as it is for 'tags'.